src/pykeen/training/training_loop.py

```diff
@@ -131,6 +133,12 @@ class TrainingLoop(Generic[BatchType], ABC):
 
     supports_slicing: ClassVar[bool] = False
 
+    @update_docstring_with_resolver_keys(
+        ResolverKey("optimizer", "class_resolver.contrib.torch.optimizer_resolver"),
+        ResolverKey("lr_scheduler", "class_resolver.contrib.torch.lr_scheduler_resolver"),
+        ResolverKey("result_tracker", "pykeen.trackers.tracker_resolver"),
```

Comment:
Adding some of these resolver keys could also be moved to a separate MR for better separation.

Reply:
i think they're fine here, don't go through the hassle